lib/bolt/cli.rb

```diff
@@ -17,6 +17,7 @@
 require 'bolt/inventory'
 require 'bolt/rerun'
 require 'bolt/logger'
+require 'bolt/module_installer'
```

Comment:
This seems unnecessary - can you just `git checkout puppet/main -- lib/bolt/cli.rb`?